frontend/public/components/deploy-image.tsx

```diff
@@ -433,6 +445,7 @@ export type DeployImageState = {
   loading: boolean,
   isi?: any,
   name: string,
+  labels: any,
```

Comment:
We should change `,` to `;` in the type declaration

```suggestion
  labels: string[];
```